src/index.html

```diff
@@ -2,26 +2,447 @@
+      </article>
+
+      <article class="card">
+        <div class="card__img-cover">
+          <img
+            src="/images/imac.jpeg"
+            alt="imac"
+            class="card__img"
+          />
+        </div>
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__descr">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="review-descr">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="price-text">Price:</p>
+
+          <p class="price-amount">$2,199</p>
+        </div>
+
+        <a class="card__button">BUY</a>
+      </article>
+
+      <article class="card">
+        <div class="card__img-cover">
+          <img
+            src="/images/imac.jpeg"
+            alt="imac"
+            class="card__img"
+          />
+        </div>
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__descr">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="review-descr">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="price-text">Price:</p>
+
+          <p class="price-amount">$2,199</p>
+        </div>
+
+        <a class="card__button">BUY</a>
+      </article>
+
+      <article class="card">
+        <div class="card__img-cover">
+          <img
+            src="/images/imac.jpeg"
+            alt="imac"
+            class="card__img"
+          />
+        </div>
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__descr">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="review-descr">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="price-text">Price:</p>
+
+          <p class="price-amount">$2,199</p>
+        </div>
+
+        <a class="card__button">BUY</a>
+      </article>
```

Comment:
The content within the `article` elements is highly repetitive. Consider using a templating engine or JavaScript to dynamically generate this content to reduce redundancy and improve maintainability.